lib/Test2/Bundle/Extended.pm

```diff
@@ -184,40 +184,48 @@ extensively to test L<Test2::Suite> itself.
 
 =head1 RESOLVING CONFLICTS WITH MOOSE
 
-    use Test2::Bundle::Extended '!meta';
+    use Test2::Bundle::Extended qw/:v1 !meta/;
 
 L<Moose> and L<Test2::Bundle::Extended> both export very different C<meta()>
 subs. Adding C<'!meta'> to the import args will prevent the sub from being
 imported. This bundle also exports the sub under the name C<meta_check()> so
 you can use that spelling as an alternative.
 
-=head2 TAGS
+=head1 EXPORT PINS
 
-=over 4
+B<The current pin used by all of Test::Suite is C<v2>.>
 
-=item :v1
+Export pins are how L<Test2::Suite> manages changes that could break backwords
+compatability. If we need to break backwards compatability we will do so by
+releasing a new pin. Old pins will continue to import the old functionality
+while new pins will import the new functionality.
 
-=item :DEFAULT
+There are several ways to specify a pin:
 
-The following are all identical:
+    # Import all the defaults provided by the 'v2' pin
+    use Package ':v2';
 
-    use Test2::Bundle::Extended;
+    # Import foo, bar, and baz deom the v2 pin.
+    use Package '+v2' => [qw/foo bar baz/];
 
-    use Test2::Bundle::Extended ':v1';
+    # Import 'foo' from the v2 pin, and import 'bar' and 'baz' from the v1 pin
+    use Package qw/+v2 foo +v1 bar baz/;
```

Comment:
Chad and I discussed this feature in Slack. IMO this introduces a maintenance nightmare in the coming years. What it implies is all versions of all features must work with all versions of all other features. For example...

    use Test2::Bundle::Extended "+v1" => "is", "+v7" => "hyperthing";

    is $thing, hyperthing;

The mind boggles at the testing matrix this introduces. Rather than versions giving us a clean break from old mistakes and bad ideas, we have to continue to account for those mistakes and ensure they work with all the new ones.

Nightmare.

I'm also not sure what the use case for this is. Generally you leave a package at a particular version. You write new code with the new version. And later you upgrade old ones as necessary.

If this *were* to be used it creates a nightmare for the test maintainers. Anyone working on this test must know detailed differences between versions of different functions. Why did they decide to use the v1 `is`? How is it different from the v7 `is` I'm used to? Nightmare.

Instead, I propose the opposite. You can't use two versions of the same package in the same namespace. If you do you get a warning. This eliminates a nightmare for us. And it eliminates a feature that will create nightmares for our users.